content/webapp/pages/homepage.tsx

```diff
@@ -31,6 +30,10 @@ import { getServerData } from '@weco/common/server-data';
 import ExhibitionsAndEvents from '../components/ExhibitionsAndEvents/ExhibitionsAndEvents';
 import CardGrid from '../components/CardGrid/CardGrid';
 import { articleLd } from '../services/prismic/transformers/json-ld';
+import { createClient } from 'services/prismic/fetch';
```

Comment:
we've got a mix of relative and absolute paths here (`../services` vs. `services) – I think we've gone with relative elsewhere, but not sure if there's a reason to favour one over the other?

Reply:
Nor do I; this was unintentional. I let VS Code handle the imports, and I guess it prefers the absolute paths? I'll fix this.

Reply:
oh yeah, VS Code is not to be trusted for its auto-imports!